Assets/Code/Interactables/Effects/Dynamic Effects/Model/DynamicEffect.cs

```diff
@@ -1,10 +1,9 @@
 ﻿using System;
-using System.Collections.Generic;
 
-public abstract class PlayEffect {
-    // This class is to be instantiated for specific effects
-    // PlayEffects will be pushed into the EffectController's queue and resolved in sequence
-    // PlayEffects can be owned by many different classes, but all effects will still be sent to the same EffectController
+public abstract class DynamicEffect {
```

Comment:
effect no longer looks like a word

Reply:
lmaoo

Reply:
I can't think of a better word to describe them though

Reply:
nah i think it's perfect xD brain go melt tho